app/views/backend/course.erb

```diff
@@ -15,6 +15,7 @@
                 <% @course.chapters.each do |chapter| %>
                    <li>
                     <%= chapter.name %>
+                    <a href="/chapter/delete/<%= chapter.id %>" <i class="material-icons">delete</i></a><br>
```

Comment:
Check nested lists! One List to list the chapters name and a nested list inside each chapter to set the description.  One solution could be:
´´´html
<ul>
  <li>Chapter name</li>
    <ul>
    <li>Description</li>
    </ul>
  </li>
  <li>Chapter name</li>
</ul>
´´´
Or use divs instead!